### PR TITLE
added initial settings for maximal resolutions for both cameras

### PR DESCRIPTION
**Pull request by alinelena**
for the moment only one resolution for both front and rear... 
still setting the resolution fails which requires more investigation
Alin

### CODE REVIEW

**Comment by sledges** [comment]
LGTM

**Comment by sledges** [comment]
@alinelena, please rename your commits into a short and consise message. You can explain in detail in commit description. So in commit window it looks like:

```
commit message goes here

Commit description is longer and it goes here.
```

**Comment by sledges** [comment]
LGTM

**Comment by foolab** [comment]
why are you adding only 1 resolution?

Could you also please squash the commits info one?

**Comment by alinelena** [comment]
I will add more resolutions once we iron more bugs...

File: data/sailfish/hammerhead/resolutions.ini [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+#[image-micro1-4:3]
+#capture = 320x240
+#viewfinder=320x240
+#preview=960x720
+#fps=15
+#night = -1
+#device=0
+#mode=image
+
+[image-low-3:2]
```
Comment (sledges) [inline data/sailfish/hammerhead/resolutions.ini:127]: I see two duplicate resolutions under this aspect ratio. If i comment this block - both disappear

Comment (sledges) [inline data/sailfish/hammerhead/resolutions.ini:127]: Duplicates are also quite often elsewhere, in pairs

**Comment by alinelena** [comment]
So the root cause seems to be the fact that camera plus cummulates the resolutions for both devices (front/rear) in the resolution section rather than the resolution of the current selected device.
So I will say this is a feature... or a bug.. since if I select the front camera only few aspect ration appear and no resolution for them.
Alin

**Comment by alinelena** [comment]
As a numpty I had the wrong device for the front camera, hence the duplication, that shall be fixed now.

**Comment by sledges** [comment]
Duplicates gone! \o/ Thanks

But now I have no resolutions written out under 3:2 in setting and when chosen (is it because it's the only mode?)
And there's still ?:? resolution with 0.1 and 0 MP.
How do we go about cleaning up those?

**Comment by alinelena** [comment]
Yes the ones with only one resolution will do not show the resolution in the settings only once selected.

the ?:? are for 12:10 ratio... we can comment them in the resolutions.ini and they will be gone...

regards,
Alin

**Comment by sledges** [comment]
Let's do that and go

**Comment by alinelena** [comment]
Done no more ?:?

Alin

**Comment by sledges** [comment]
LGTM clean and neat, ready to merge @foolab

**Comment by foolab** [comment]
obsoleted by the caps resolutions

**Pull request closed without merging**